internal/workos/http_test.go

```diff
@@ -1,18 +1,37 @@
 package workos
 
 import (
+	"github.com/stretchr/testify/require"
```

Comment:
nit: Not sure if this was a `gofmt` change or a manual one, but AFAIK it's idiomatic to keep the imports for the standard library separate from external imports.